spec/amp-actions-and-events.md

```diff
@@ -324,7 +322,8 @@ Including: `input[type=radio]`, `input[type=checkbox]`, `input[type=range]`, and
   </tr>
 </table>
 
-## `AMP` target
+## Special targets
```

Comment:
I just amended my existing commit and did a force push because the new changes can still be considered in-scope of the existing commit. I can see the changes [here](https://github.com/ampproject/amphtml/pull/9568/files):

I have the habit of not polluting the commit history with intermediate commits (that are addressing code review comments, for example) as I remember some people are pretty keen about this. Now, I haven't used Github for a while and maybe that's not recommended anymore (maybe it disrupts code reviews somehow)?